docs/TypeScript.md

```diff
@@ -204,65 +204,94 @@ When updating core types you should make a PR to this repository. Ensure you:
 <a id="plugin-types"></a>
 ### Plugin Types
 
-Typings for plugins are hosted in DefinitelyTyped. This means when using plugins you should install like so:
+Plugins maintained by and orginized under the fastify organization on GitHub should ship with typings just like fastify itself does.
+Some plugins already include typings but many do not. We are happy to accept contributions to those plugins without any typings.
+
+Typings for third-party-plugins may either be included with the plugin or hosted on DefinitelyTyped. Remember, if you author a plugin to either include typings or publish them on DefinitelyTyped!
+
+Install a plugin with typings hosted on DefinitelyTyped like so:
 
 ```
-npm install fastify-url-data @types/fastify-url-data
+npm install fastify-url-data && npm install -D @types/fastify-url-data
```

Comment:
fastify-url-data ships typings within the package itself, these were added in Sept 2018 https://github.com/fastify/fastify-url-data/pull/8.

I'm ok in removing this example altogether, this should be business as usual stuff for TypeScript users. We might want to add a link to the DefinitelyTyped docs maybe.

Reply:
Yeah, I'd say removing this doesn't take away anything from the docs.